README.md

```diff
@@ -23,6 +23,10 @@ Shows your week of clocking at the top of Central and prompts you to enter some
 
 ![screen shot 2017-08-16 at 10 04 53 am](https://user-images.githubusercontent.com/430385/29367332-8beceac6-826a-11e7-9e62-3800663c5b22.png)
 
+## `dotorg-helper.js`
```

Comment:
I didn't think of this on the previous script you wrote, but could you rename the file to `dotorg-helper.user.js`? That way Tampermonkey will prompt the user to install the script when they view the raw source in their browser.